.github/workflows/openapi-check.yml

```
@@ -47,7 +47,28 @@
           # Start server in background to fetch spec from API endpoint
           MIX_ENV=test mix phx.server &
           SERVER_PID=$!
-          sleep 15  # Wait for server to start
+          
+          # Wait for server to be ready with retries
+          echo "Waiting for Phoenix server to start..."
+          max_attempts=30
+          attempt=0
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)